frontend/src/env.ts

```diff
@@ -104,3 +104,7 @@ export const IS_PRODUCTION: boolean =
 export const STRIPE_MAX_PAYMENT = 999999.99
 // Flathub enforces a minimum payment amount of $1 to cover fees
 export const FLATHUB_MIN_PAYMENT = 1
+
+export const APP_BASE_URL: string = `${BASE_URI}/app`
+
+export const APP_VERIFICATION_STATUS = (id: string): string => `${BASE_URI}/${id}/get_verification_status`
```

Comment:
This is what's causing tests to fail. Could you add a new line at the beginning of the template literal and the end of the arrow function?

Reply:
That won't fix the tests, it also needs to be called correctly